packages/pytest-simcore/src/pytest_simcore/helpers/utils_tokens.py

```diff
@@ -50,7 +49,11 @@ async def get_token_from_db(
             token_service=token_service,
             token_data=token_data,
         )
-        stmt = sa.select([tokens,]).where(expr)
+        stmt = sa.select(
+            [
+                tokens,
```

Comment:
remove coma and reformat
```suggestion
                tokens
```